src/controllers/RatingController.js

```diff
@@ -0,0 +1,43 @@
+import status from '../config/status';
+
+import { Article } from '../queries';
+
+/**
+ * RatingController: Class that handles article rating
+ */
+class RatingController {
+  /**
+   * @param {object} req Request sent to the route
+   * @param {object} res Response from server
+   * @returns {object} Object representing the response returned
+   */
+  static async create(req, res) {
+    const data = {
+      rating: req.body.rating,
+      articleId: req.article.id,
+      userId: req.user.id
+    };
+    const response = await Article.rate.create(data);
+    return res.status(response.statusCode).send({
+      rating: {
+        message:
+          response.statusCode === status.OK
+            ? 'Your article rating has been updated'
```

Comment:
the logic here seems flawed if the status is ok `article update successful` else even in the case of an error the message displayed is `Thank you for rating this article`